prow/cmd/peribolos/main.go

```diff
@@ -80,6 +81,7 @@ func (o *options) parseArgs(flags *flag.FlagSet, args []string) error {
 	flags.IntVar(&o.tokensPerHour, "tokens", defaultTokens, "Throttle hourly token consumption (0 to disable)")
 	flags.IntVar(&o.tokenBurst, "token-burst", defaultBurst, "Allow consuming a subset of hourly tokens in a short burst")
 	flags.StringVar(&o.dump, "dump", "", "Output current config of this org if set")
+	flags.BoolVar(&o.ignorePrivateTeams, "ignore-private-teams", false, "Do not dump private teams if set")
```

Comment:
Won't this cause peribolos to then delete the private teams once it starts managing teams?